alacritty.yml

```diff
@@ -58,17 +58,17 @@ scrolling:
 font:
   # The normal (roman) font face to use.
   normal:
-    family: Berkeley Mono
+    family: BerkeleyMono Nerd Font Mono
     style: Regular
 
   # The bold font face
   bold:
-    family: Berkeley Mono
+    family: BerkeleyMono Nerd Font Mono
     style: Bold
 
   # The italic font face
   italic:
-    family: Berkeley Mono
+    family: BerkeleyMono Nerd Font Mono
```

Comment:
:sunglasses:

Reply:
@mrnugget your comment about "emoji" changed my mind :) There are some icons I dislike, but I'll try it out at least for couple of weeks.

Reply:
Yeah, agree. It somehow also doesn't work as well on macOS as on Linux for me.